### DISCUSSION TITLE
Where is PK analysis button in MoBi ?

### DISCUSSION THREAD

**Discussion by Microbiotman** [Open-Systems-Pharmacology/Forum#1547] [Questions & Problems] [answered]
Hi everybody!

I didn't find the button for PK analysis in MOBI. Does that mean that I have to calculate outputs like AUC on my own ?
Thank you very much

**Comment by StephanSchaller** [comment, marked as answer]
Only available in PK.Sim for now but a feature wish for MoBi for the future.
